Jenkinsfile

```diff
@@ -185,10 +185,10 @@ pipeline {
           stage('Initilize qemu') {
             steps {
               sh """
-                free -h && df -h
-                cat /proc/cpuinfo
-                # initialize qemu
-                docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+                  free -h && df -h
+                  cat /proc/cpuinfo
```

Comment:
lscpu? less crowded

Reply:
We can clear that up in a follow up, I actually didn't mean to reindent these lines, so I've undone the change